frontend/components/pages/CourseList.tsx

```diff
@@ -40,41 +42,56 @@ export function CourseList({ courses, handleTextChange }: CourseListProps) {
 
   return (
     <Center>
-      <VStack mx="auto" padding={'60px 96px'}>
-        <Heading py={10} color={PRIMARY_FONT_COLOR} fontSize="36px">
-          コース一覧
-        </Heading>
-        <Box marginLeft={'auto'}>
-          <InputGroup>
-            <Input
-              value={text}
-              onChange={(e) => handleInputChange(e)}
-              onKeyDown={(e) => handleKeyDown(e)}
-              variant="outline"
-              placeholder="javaScript"
-              background={'white'}
-            />
-            <InputRightElement onClick={() => handleSubmit()}>
-              <SearchIcon color="gray.500" />
-            </InputRightElement>
-          </InputGroup>
-        </Box>
-        <Box mb={10} minW={'1024px'}>
-          <Flex
-            alignItems="center"
-            pb={2}
-            borderBottom="1px"
-            borderColor={'blackAlpha.500'}
-          ></Flex>
-          {courses.length === 0 && (
-            <VStack mt="40px">
-              <Heading color={PRIMARY_FONT_COLOR} fontSize="36px">
-                検索に一致するコースはありませんでした
-              </Heading>
-            </VStack>
-          )}
-          {courses && <CourseCard courses={courses} />}
-        </Box>
+      <VStack padding={'60px 20px'}>
+        <Flex>
+          <Box mb={10} minW={'880px'}>
+            <Heading
+              py={10}
+              color={PRIMARY_FONT_COLOR}
+              fontSize="36px"
+              textAlign="center"
+            >
+              コース一覧
+            </Heading>
+            <Flex justifyContent={'flex-end'} mb={4}>
+              <InputGroup width="300px">
+                <Input
+                  value={text}
+                  onChange={(e) => handleInputChange(e)}
+                  onKeyDown={(e) => handleKeyDown(e)}
+                  variant="outline"
+                  placeholder="javaScript"
+                  background={'white'}
+                />
+                <InputRightElement onClick={() => handleSubmit()}>
+                  <SearchIcon color="gray.500" />
+                </InputRightElement>
+              </InputGroup>
+            </Flex>
+            <Flex
+              alignItems="center"
+              pb={2}
+              borderBottom="1px"
+              borderColor={'blackAlpha.500'}
+            ></Flex>
+            {courses.length === 0 && (
+              <VStack mt="40px">
+                <Heading color={PRIMARY_FONT_COLOR} fontSize="36px">
+                  検索に一致するコースはありませんでした
+                </Heading>
+              </VStack>
+            )}
+            {courses && <CourseCard courses={courses} />}
+          </Box>
+          <VStack>
+            <Box>
+              <LineBanner />
+            </Box>
+            <Box>
+              <AchievementBanner />
+            </Box>
+          </VStack>
```

Comment:
```suggestion
          <VStack margin={ここを変更}>
            <Box>
              <LineBanner />
            </Box>
            <Box>
              <AchievementBanner />
            </Box>
          </VStack>
```

全体のブロックに対して幅を決めるようにしてもらえると！